client/src/pages/Landing/Landing.js

```diff
@@ -56,32 +52,42 @@ const Landing = () => {
     */
   const dispatch = useDispatch()
   const errorWrap = useErrorWrap()
+  const config = {
+    responseType: 'id_token',
+    expoClientId: Constants.manifest.extra.expoClientId,
+    iosClientId: Constants.manifest.extra.iosClientId,
+    androidClientId: Constants.manifest.extra.androidClientId,
+    scopes: ['profile', 'email'],
+  }
   const [quote] = useState(`${i18n.t('dialogue.landingQuote')}`)
+  const [, response, promptAsync] = Google.useAuthRequest(config, {
+    useProxy: true,
+  })
 
-  const loginUser = async () => {
-    await errorWrap(async () => {
-      const config = {
-        iosClientId: Constants.manifest.extra.iosClientId,
-        androidClientId: Constants.manifest.extra.androidClientId,
-      }
-      const { idToken, refreshToken } = await Google.logInAsync(config)
-      const guid = Google.getPlatformGUID(config)
-      const clientId = `${guid}.apps.googleusercontent.com`
-      if (idToken !== undefined && refreshToken !== undefined) {
-        const userData = {
-          idToken,
+  useEffect(() => {
+    errorWrap(async () => {
+      if (response?.type === 'success') {
+        const { id_token: idToken } = response.params
+        if (idToken !== undefined) {
+          const userData = {
+            idToken,
+          }
+          // call API
+          await saveUserIDToken(idToken)
+          await createUser(userData)
+          // Save to Secure Store
+
+          // TODO: Add back support for Refresh Tokens
+
+          // await saveUserRefreshToken(refreshToken)
+          // await saveUserClientId(clientId)
```

Comment:
this looks like it may come up again in the auth refresh fix, but if it doesn't we should probably remove commented out code